### PR TITLE
weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -218,7 +218,7 @@
       ],
       "multiArchVersions": [
         "1.10.5",
-        "1.10.5.3"
```
Comment (andyliuliming) [inline vhdbuilder/packer/components.json:221]: the version we are currently using is 1.10.5 or 1.10.5.3?

File: vhdbuilder/packer/kube-proxy-images.json [hunk 2]
```diff
@@ -5,19 +5,20 @@
...
-          "1.21.7-hotfix.20220130",
+          "1.20.15-hotfix.20220201.1",
           "1.21.7-hotfix.20220204",
-          "1.21.9-hotfix.20220130",
+          "1.21.7-hotfix.20220310.1",
           "1.21.9-hotfix.20220204",
+          "1.21.9-hotfix.20220310.1",
           "1.22.4-hotfix.20220201",
-          "1.22.4-hotfix.20220223",
+          "1.22.4-hotfix.20220310.1",
```
Comment (andyliuliming) [inline vhdbuilder/packer/kube-proxy-images.json:17]: what's the version currently using?
1.22.4-hotfix.20220201
or 
1.22.4-hotfix.20220223

Comment (andyliuliming) [reply vhdbuilder/packer/kube-proxy-images.json:17]: and for other versions.

**Review by andyliuliming — CHANGES_REQUESTED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (HDYA) [reply vhdbuilder/packer/components.json:221]:
```
{{- else if eq .component "cilium-agent" -}}
1.10.5
{{- else if eq .component "cilium-operator" -}}
1.10.5
```

File: vhdbuilder/packer/kube-proxy-images.json (hunk 2 above)
Comment (HDYA) [reply vhdbuilder/packer/kube-proxy-images.json:17]: the former, and confirmed removed versions are not in use
```
		"v1.20.13": "v1.20.13-hotfix.20220210",
		"v1.20.15": "v1.20.15-hotfix.20220201",
		"v1.21.7":  "v1.21.7-hotfix.20220204",
		"v1.21.9":  "v1.21.9-hotfix.20220204",
		"v1.22.4":  "v1.22.4-hotfix.20220201",
		"v1.22.6":  "v1.22.6-hotfix.20220130",
		"v1.23.3":  "v1.23.3-hotfix.20220130.2",
```

**Review by andyliuliming — APPROVED** [review]

**Pull request merged**